package/CHANGELOG

```diff
@@ -25,6 +25,8 @@ Fixes
 Enhancements
   * LAMMPSDump Reader use of `continuous` option to stitch trajectories
     (Issue #3546)
+  * Add support for reading chainID info from prmtop amber topologies
+	(PR #4007)
```

Comment:
indentation

Reply:
And add your GitHub handle to the contributor line after  MohitKumar020291

Reply:
Also add that the TOPParser now properly recognizes (and ignores) %COMMENT lines.

Reply:
Done.